app/src/main/java/com/example/ncov19traking/ui/graphs/GraphsFragment.kt

```diff
@@ -56,10 +60,12 @@ class GraphsFragment : Fragment() {
                         ContextCompat.getColor(root.context, R.color.recoveredColor)
                     )
                 )
+                val lineData = LineData(allLineData)
```

Comment:
you don't need this variable here you can directly assign `LineData(allLineData)` to `chart.data`, and is short enough to not be a mess hahaha

Reply:
fixed